SubsoniciousKit/SubsoniciousKit/Player/SubsonicPlayerItem.swift

```diff
@@ -0,0 +1,19 @@
+//
+//  SubsonicPlayerItem.swift
+//  SubsoniciousKit
+//
+//  Created by Bilal on 21/08/2020.
+//  Copyright © 2020 Bilal Benlarbi. All rights reserved.
+//
+
+import Foundation
+import MediaPlayer
+
+final class SubsonicPlayerItem: AVPlayerItem {
+    let subsonicId: String
+
+    init(subsonicId: String, asset: AVAsset, automaticallyLoadedAssetKeys: [String]? = nil) {
+        self.subsonicId = subsonicId
+        super.init(asset: asset, automaticallyLoadedAssetKeys: automaticallyLoadedAssetKeys)
```

Comment:
what is these `automaticallyLoadedAssetKeys` couldn't be an enum  ?

Reply:
these are from the `init(asset:automaticallyLoadedAssetKeys:)` of the superclass `AVPlayerItem`. I used the same type used by the super init :)